br/pkg/lightning/backend/kv/kv2sql.go

```diff
@@ -88,10 +88,16 @@ func (t *TableKVDecoder) IterRawIndexKeys(h kv.Handle, rawRow []byte, fn func([]
 	}
 
 	indices := t.tbl.Indices()
+	isCommonHandle := t.tbl.Meta().IsCommonHandle
 
 	var buffer []types.Datum
 	var indexBuffer []byte
 	for _, index := range indices {
+		// skip clustered PK
+		if index.Meta().Primary && isCommonHandle {
```

Comment:
should we skip `clustered & single-int column` kind of pk?

Reply:
added test https://github.com/pingcap/tidb/pull/44799/commits/7338cc1f191a019bbad29df2ab63d51708cf8eb5

When PKIsHandle (test file line 83), I remember the PK will not appear in `indices`, so no need to skip it.